### PR TITLE
fix seed in `GraphNorm` tests

### PR DESCRIPTION
**Pull request by puririshi98**
solves https://github.com/pyg-team/pytorch_geometric/issues/6622

### CODE REVIEW

File: test/nn/norm/test_graph_norm.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import torch
 
+from torch_geometric import seed_everything
 from torch_geometric.nn import GraphNorm
 from torch_geometric.testing import is_full_test
 
 
 def test_graph_norm():
+    seed_everything(42)
```
Comment (rusty1s) [inline test/nn/norm/test_graph_norm.py:9]:
```suggestion
from torch_geometric.nn import GraphNorm
from torch_geometric.testing import is_full_test


def test_graph_norm():
    torch.manual_seed(42)
```

**Review by rusty1s — APPROVED** [review]
